### PR TITLE
OMI-498 restructre documentation

### PR DESCRIPTION
**Pull request by cereniyim**

### CODE REVIEW

**Review by bernardolk — APPROVED** [review]

File: documentation/Spec2Vec.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+The supported metadata keys for omigami are (case insensitive):
+- "smiles",
+- "compound_name",
+- "instrument",
+- "parent_mass",
+- "inchikey_smiles",
+- "inchikey_inchi"
+- "precursor_mz" or "pepmass"
+
```
Comment (carloszanella) [inline documentation/Spec2Vec.md]: This can be removed

File: documentation/Spec2Vec.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ion_mode = "positive"  # either positive or negative
+
+result = client.match_spectra(
+    mgf_file_path, n_best_matches, ion_mode,
+)
+
+# Spectra is sent in batches to the predictor. 
+# If any of the batches fail, the following command will return the list of spectra in the failed batch. 
+# The problem may have been caused by just one or more spectrum inside the failed batch.
+failed_spectra = client.failed_spectra()
```
Comment (carloszanella) [inline documentation/Spec2Vec.md]: this is a property not a method

File: documentation/Spec2Vec.md (hunk 1 above)
Comment (carloszanella) [reply documentation/Spec2Vec.md]: same for MS2DS file

Comment (cereniyim) [reply documentation/Spec2Vec.md]: ok I will remove

**Review by carloszanella — APPROVED** [review]

**Pull request merged**